docs/ts/primitives/services.mdx

```diff
@@ -25,4 +25,29 @@ export default new Service("my-service");
 
 That's it! Encore will consider this directory and all its subdirectories as part of the service.
 
-For more on how to structure your application, see the [app structure guide](/docs/ts/develop/app-structure).
\ No newline at end of file
+For more on how to structure your application, see the [app structure guide](/docs/ts/develop/app-structure).
+
+## Defining services implicitly
```

Comment:
I think we should describe this way of creating services as well as current users probably have this setup. We might also have some examples floating around defining services in this way. Thoughts?